client/src/theme/styles/components/frontend/project/_resource-thumbnail-list.scss

```diff
@@ -1,30 +1,57 @@
 .resource-thumbnail-list {
   ul {
     @include listUnstyled;
+
+    // flex fallback
     display: flex;
     flex-wrap: wrap;
+    // push it 20px to the left to negate the left padding on the resources
     margin-left: -20px;
-  }
 
-  li {
-    display: flex;
-    width: 50%;
-    padding-left: 20px;
-    margin-bottom: 20px;
-
-    &:nth-last-of-type(-n + 2) {
-      display: none;
+    li {
+      // 2 columns
+      width: 50%;
+      // 20px padding on the bottom and left
+      padding: 0 0 20px 20px;
+      @include respond($break65) {
+        // 4 columns
+        width: 25%;
+      }
+      @include respond($break90) {
+        // 5 columns
+        width: 20%;
+      }
     }
 
-    @include respond($break65) {
-      width: 25%;
-    }
+    // grid style if the browser supports it
+    @supports ( grid-area: auto ) {
+      display: grid;
+      // use grid gap instead
+      grid-gap: 20px;
```

Comment:
The margin below the list was lost switching to grid, so make sure to add it back. Otherwise the spacing beneath resources and resource collections becomes inconsistent.

![Screen Shot 2019-05-01 at 9 27 13 AM copy](https://user-images.githubusercontent.com/12454203/57028907-2b5a2b80-6bf5-11e9-9306-45054db039b5.png)

Reply:
Good catch. While I was looking around at that I also found the grid was clipping the shadows on the left side. Fixed that as well.

Reply:
Good catch of yours too! It looks like that padding isn't quite sufficient to prevent clipping the shadow on the bottom of the items though.

![Screen Shot 2019-05-01 at 4 15 15 PM](https://user-images.githubusercontent.com/12454203/57049374-b1de2f80-6c2c-11e9-8e81-39a433993d33.png)

Reply:
I can't easily keep the grid from clipping the shadows without pushing the `Total Resources` tag further away than the 50px at which it currently sits. I could squeeze a little more space if I remove the top margin and padding on the total resources, but that would require a custom rule for that one element.

Reply:
And I'm seeing that there's clipping on the righthand side as well.

You're right that we should avoid writing one-off rules for different elements. Which leads me to think that in spite of very cool technique to hide overflowing grid items, we probably need to fallback to the other solution you proposed originally, of just dropping that 4-column intermediate layout. The shadow here just doesn't play well with `overflow: hidden`, and I don't think it's worth the CSS workarounds to resolve it.

Reply:
No worries. Often times the simplest solution is the best. That grid would have worked great if not for the shadows.

Reply:
Yeah exactly. And it's easy to miss one (or more) of the visual states an element might have before creating a solution.